packages/oauth/src/openshift/osoauth-service.ts

```diff
@@ -283,14 +283,13 @@ export class OSOAuthService implements OAuthProtoService {
   private doLogout(config: OpenShiftOAuthConfig): void {
     this.fetchUnregister?.()
 
-    const currentURI = new URL(window.location.href)
     // The following request returns 403 when delegated authentication with an
     // OAuthClient is used, as possible scopes do not grant permissions to access the OAuth API:
     // See https://github.com/openshift/origin/issues/7011
     //
     // So little point in trying to delete the token. Lets do in client-side only
     //
-    forceRelogin(currentURI, config)
+    forceRelogin(new URL(`${window.location.origin}/${hawtio.getBasePath()}`), config)
```

Comment:
Shouldn't it lead to `https://host:port/undefined` if it doesn't handle undefined correctly?  I suggest to declare the URL in a separate line just like the original code.